demisto_sdk/scripts/init_validation_script.py

```diff
@@ -0,0 +1,540 @@
+                )
+            )
+        if not validator_class_name.endswith("Validator"):
+            validator_class_name = f"{validator_class_name}Validator"
+        self.class_declaration = (
+            f"class {validator_class_name}(BaseValidator[ContentTypes]):"
+        )
+
+    def get_git_statuses(self):
+        """
+        Request the supported git statuses and ensure the input is valid.
+        """
+        self.git_statuses_str = str(
+            input(
+                "Enter a comma separated list of git statuses the validation should run on,\n"
+                "R: renamed files\nA: added files\nD: deleted files\nM: modified files\nor leave empty if you wish that the validation will run on all files: "
+            )
+        )
+        while self.git_statuses_str and not set(
+            self.git_statuses_str.split(",")
+        ).issubset({"A", "R", "M", "D"}):
+            self.git_statuses_str = str(
+                input(
+                    "Please make sure to insert either valid inputs which are:\n"
+                    "R: renamed files\nA: added files\nD: deleted files\nM: modified files\nor leave empty if you wish that the validation will run on all files: "
+                )
+            )
+
+    def get_content_types(self):
+        """
+        Request the supported content types list and ensure the input is valid.
+        """
+        supported_content_types = "\n".join(
+            [
+                f"{key}: {value.get('content_type')}"
+                for key, value in CONTENT_TYPES_DICT.items()
+            ]
+        )
+        content_types = str(
+            input(
+                f"""Enter a comma separated list of content types to be supported by the validation.
+The supported types are:
+{supported_content_types}
+Fill the content types as the numbers they appear as: """
+            )
+        )
+        while not self.is_valid_content_types_input(content_types):
+            if not content_types:
+                content_types = str(
+                    input("Please make sure to enter at least one content type: ")
+                )
+            else:
+                content_types = str(
+                    input(
+                        f"Please make sure all content types are valid integers between between {self.min_content_type_val} and {self.max_content_type_val}: "
+                    )
+                )
+        self.content_types = content_types.split(",")
+
+    def is_valid_content_types_input(self, content_types: str) -> bool:
+        """Validate that the content types input is valid (at least one content type, and all inputs are in the content types range.)
+
+        Args:
+            content_types (str): comma separated list of the content_types input.
+
+        Returns:
+            bool: True if the input is valid, otherwise return False.
+        """
+        return bool(content_types) and all(
```

Comment:
```suggestion
        return content_types and all(
```

Reply:
getting type errors when doing it this way